src/main/resources/application.conf

```diff
@@ -298,7 +298,7 @@ mantis {
 
   sync {
     # Whether to enable fast-sync
-    do-fast-sync = true
+    do-fast-sync = false
```

Comment:
Minor detail: Why was this value changed?

Reply:
Sorry about that, I commited my local changes again because I added some params to the config.. =(